src/components/book.js

```diff
@@ -1,24 +1,25 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/prop-types */
```

Comment:
As mentioned in the previous comment and for the same reasons, kindly avoid modifying them.

This has been repeated in many other files, please modify them too.